core/eval.go

```diff
@@ -856,6 +857,97 @@ func evalQWATCH(args []string, c *Client) []byte {
 	return RESP_OK
 }
 
+func evalBITCOUNT(args []string) []byte {
+	var err error
+
+	// if more than 4 arguments are provided, return error
+	if len(args) > 4 {
+		return Encode(errors.New("ERR invalid number of arguments for `evalBITCOUNT` command"), false)
```

Comment:
Let's ensure that the errors are exactly what we see in Redis 7.2.5. Also, it is a bad idea to expose the function name in the error message.